google/cloud/forseti/scanner/scanners/iam_rules_scanner.py

```diff
@@ -170,10 +170,9 @@ def _output_results(self, all_violations, resource_counts):
         if self.scanner_configs.get('output_path'):
             LOGGER.info('Writing violations to csv...')
             output_csv_name = None
-            with csv_writer.write_csv(
-                resource_name=resource_name,
-                data=all_violations,
-                write_header=True) as csv_file:
+            with csv_writer.write_csv(resource_name=resource_name,
```

Comment:
Ditto as above, why this change?

Reply:
The visual line indent was the issue I believe here: https://www.python.org/dev/peps/pep-0008/#indentation